config/jobs/kubernetes-sigs/karpenter/OWNERS

```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- ellistarn
+- jonathan-innis
+- tzneal
+- bwagner5
+- njtran
+- jackfrancis
+reviewers:
+- jackfrancis
```

Comment:
Don't think that you need to also be listed as a reviewer if you are an approver